sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -489,7 +615,7 @@ enum class matrix_type {
   sint4,
```

Comment:
What are `sint2` and `sint4`?  I don't see any description above telling how to use these types in a joint matrix.  Same for `uint2` and `uint4`.

Reply:
These have not been added yet.
@JackAKirk are these types (2 bits, 4 bits) supported on the CUDA backend?

Reply:
> These have not been added yet. @JackAKirk are these types (2 bits, 4 bits) supported on the CUDA backend?

No they won't be supported.

2 bit is unsupported on Nvidia hardware. 4 bit is supported via cuda wmma at the moment, but it is deprecated, and unsupported in sm90, so we didn't plan on adding it. It will remain supported by `mma` but that will only potentially be used in libraries. The only missing type from `joint_matrix` for the cuda backend that we may add in the future is the single-bit wmma.

Reply:
@gmlueck @JackAKirk, should I just remove them for now? Once we actually add support for them, I can put them back here.